wagtail_localize/views/edit_translation.py

```diff
@@ -984,6 +990,8 @@ def restart_translation(request, translation, instance):
 
 
 @api_view(["PUT", "DELETE"])
+@permission_classes([IsAuthenticated])
+@authentication_classes([SessionAuthentication])
```

Comment:
The one thing I'll quickly throw into the mix here is that IIUC this overrides rather than appends, so `edit_string_translation` and `edit_override` endpoints kind of become "internal only" API endpoints. i.e: if I wanted to explicitly write a third party app that interacts with those endpoints using token auth or JWTs or whatever, I now can't do it. If we wanted to enable that use-case, we could do something like

```py
@authentication_classes([SessionAuthentication] + settings.DEFAULT_AUTHENTICATION_CLASSES)
```

I don't quite have the context to say whether that is something we would want or not, so I'll defer to your judgement but just thought I'd flag it.

Reply:
cool. Sounds sensible then :+1: